packages/misc/src/lib/misc.ts

```diff
@@ -210,16 +213,13 @@ export const throwGenericError = (e: any): never => {
 export const isNodeClientErrorV1 = (
   nodeError: NodeClientErrorV0 | NodeClientErrorV1
 ): nodeError is NodeClientErrorV1 => {
-  return (
-    nodeError.hasOwnProperty('errorCode') &&
-    nodeError.hasOwnProperty('errorKind')
-  );
+  return 'errorCode' in nodeError && 'errorKind' in nodeError;
 };
 
 export const isNodeClientErrorV0 = (
   nodeError: NodeClientErrorV0 | NodeClientErrorV1
 ): nodeError is NodeClientErrorV0 => {
-  return nodeError.hasOwnProperty('errorCode');
+  return 'nodeError' in nodeError;
```

Comment:
Should this be checking `errorCode`?  As w/ above comment, rather than ducktype it, it'd be better if wherever we throw these types of errors set the `name` to a specific value and we check that instead of duck-typing based on a single property.

Reply:
This is removed in the PR https://github.com/LIT-Protocol/js-sdk/pull/576 so it would not apply anymore
Lucky us, it was only used in `throwError`